core/src/main/java/org/apache/calcite/util/Sarg.java

```diff
@@ -131,6 +137,18 @@ public StringBuilder printTo(StringBuilder sb,
         && containsNull == ((Sarg) o).containsNull;
   }
 
+  /** Returns whether this Sarg includes all values (including or not including
+   * null). */
```

Comment:
Frankly speaking, the wording of `including or not including` is moot. What does that mean? Does that mean it ignores nulls? Does that mean it accounts for nulls?